### DISCUSSION TITLE
Try to update a Gosund SP111

### DISCUSSION THREAD

**Discussion by MyUniversalDeveloper** [arendst/Tasmota#13087] [Q&A] [answered]
Hi,

I've tried to make an update on a Gosund SP111 from Tasmota 9.5.0 to a newer version. This fails because of:
Upload Failed
Upload buffer miscompare
Is there any solution for this problem of (maybe only) mine?

Thanks for your help

**Comment by barbudor** [comment, marked as answer]
You didn't specified how you tried to update
But standard method is to use OTA using directly the url http://ota.tasmota.com/tasmota/tasmota.bin.gz
(not downloading the file, letting tasmota doing it)

All proper upgrade methods are listed in the docs https://tasmota.github.io/docs/Upgrading/#upgrade-using-webui
including the reason why you got this error message

**Reply by MyUniversalDeveloper** [reply]
Hi, thanks a lot for your quick help. In the past the OTA update doesn't work for me. But now it works fine.
